### PR TITLE
Login logout flow

### PR DESCRIPTION
**Pull request by nurmerey**
- [x] Fixed login flow
- [x] Fixed logout flow
- [x] Fixed toke creation for existing user
- [x] Fixed projects API to have access for non-auth users
- [x] Added email info to the projects request
- [x] Created a new /user url for auth only

### CODE REVIEW

File: scprojects/views.py [hunk 1]
```diff
@@ -23,22 +23,12 @@ def index(request):
...
-    if request.user.is_authenticated:
-        token = Token.objects.get(user=request.user)
-    print("auth token", str(token), request.user, request.user.is_authenticated)
-    response["user"] = {
-        "name": request.user.username,
-        "is_authenticated": request.user.is_authenticated,
-        "csrf_token": csrf.get_token(request),
-        "auth_token": str(token)
-    }
+    print("ccc request auth", request.auth)
```
Comment (partiallymisplaced) [inline scprojects/views.py]: 🗑

File: scprojects/views.py [hunk 2]
```diff
@@ -47,6 +37,31 @@ def projects(request):
     return JsonResponse(response)
 
 
+@csrf_exempt
+@api_view(['GET'])
+@authentication_classes([TokenAuthentication])
+@permission_classes([IsAuthenticated])
+def user(request):
+    response = {}
+    print("ccc request auth", request.auth)
```
Comment (partiallymisplaced) [inline scprojects/views.py]: 🗑

File: scprojects/views.py [hunk 3]
```diff
@@ -47,6 +37,31 @@ def projects(request):
...
+    token = ''
+    if request.user.is_authenticated:
+        token = Token.objects.get(user=request.user)
+        print("auth token", str(token), request.user,
```
Comment (partiallymisplaced) [inline scprojects/views.py]: 🗑

File: scprojects/views.py [hunk 4]
```diff
@@ -85,18 +100,31 @@ def add_project(request):
...
+@authentication_classes([TokenAuthentication])
+@permission_classes([IsAuthenticated])
 def login(request):
     token = ''
     if request.user.is_authenticated:
-        token = Token.objects.get(user=request.user)
-    return HttpResponseRedirect("http://localhost:3000/token/"+str(token))
+        try:
+            token = Token.objects.get(user=request.user)
+            print("Login token from DB ", token)
```
Comment (partiallymisplaced) [inline scprojects/views.py]: 🗑

File: scprojects/views.py [hunk 5]
```diff
@@ -85,18 +100,31 @@ def add_project(request):
...
+            return HttpResponseRedirect("http://localhost:3000/token/"+str(token))
+        except Token.DoesNotExist:
+            print("Token.DoesNotExist")
```
Comment (partiallymisplaced) [inline scprojects/views.py:113]: 🗑 ?

Comment (nurmerey) [reply scprojects/views.py:113]: Keeping the print on L119 for debugging purposes, but removed the one on L116

Comment (partiallymisplaced) [reply scprojects/views.py:113]: 👍

**Review by partiallymisplaced — APPROVED** [review]
LGTM 😎

**Pull request merged**